src/make_ntuples.c

```diff
@@ -81,6 +83,17 @@ int run(char * in_filename, bool use_fmt, bool debug, int nevn, int run_no, doub
         }
 
         // Process DIS event.
+        // Checking existence of trigger electron in event
+        particle p_el = particle_init();
+        for (UInt_t pos = 0; pos < rtrk.index->size(); ++pos) {
+            p_el = use_fmt ? particle_init(&rpart, &rtrk, &ftrk, pos)
+                           : particle_init(&rpart, &rtrk, pos);
+            if(p_el.is_trigger_electron) break;
```

Comment:
Initializing a whole particle just to see if it's a trigger electron is a bit much. Maybe perform this check before initializing and **then** initialize `p_el`.

Reply:
The method used now is more "economic":
1.- Inside event
2.- Check the PID and status given by reconstruction
3.- If a trigger electron is found, then save the pindex and the pos
    a .- I kept the trigger electron filter (see next issue)
4.- Obtain variables of trigger electron given by pindex and pos and write to both trees
5.- Obtain the rest of particles